packages/client/components/EmailPasswordAuthForm.tsx

```diff
@@ -117,7 +117,9 @@ const EmailPasswordAuthForm = forwardRef((props: Props, ref: any) => {
 
   const handleBlur = async (e: React.FocusEvent<HTMLInputElement>) => {
     const {name} = e.target
-    setDirtyField(name)
+    if (name === 'email' || name === 'password') {
```

Comment:
I added extra type safety to `useForm` so that `setDirtyField` and `setValue` must use values that were passed into `useForm`. One downside is  `handleBlur` is less reusable. We could use `name!` but that's error-prone, so I think checking whether it's a valid name is the best approach.